app/src/main/assets/database/CreateIndoorLocationSchemaScript.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN TRANSACTION;
```

Comment:
Why do we have sql in our code?

Reply:
I no longer see it in the code.

@makchamp can maybe shine some light to this 😄

Reply:
While this is not used anywhere in the code, I would prefer to leave it here for reference for now (as the database is not complete with all the location yet). This is what is used to generate the .db file (with the program DB browser (sqlite).